src/main/scala/com/yotpo/metorikku/metric/Metric.scala

```diff
@@ -100,6 +100,9 @@ case class Metric(configuration: Configuration, metricDir: Option[File], metricN
     log.info(s"Starting to Write results of ${dataFrameName}")
     try {
       writer.write(dataFrame)
+      //TODO: wait for the hook
```

Comment:
Ron suggests to try to use flush inestead of send in the kafka reporter, maybe it will send before the process stops